Project/GNU/CLI/autogen.sh

```diff
@@ -1,29 +1,10 @@
-#libtoolize 
-if test "$(uname)" = "Darwin" ; then
-  #Darwin based Systems like Mac OS X: libtoolize is called glibtoolize.
-  glibtoolize --automake
-else
-  libtoolize --automake
+if which autoreconf > /dev/null; then
+  rm -rf autom4te.cache
+  rm -f aclocal.m4 ltmain.sh
+  autoreconf -f -i -v --warnings=all || exit 1
 fi
-
-#aclocal
-if test -e /usr/bin/aclocal-1.11 ; then
-  #OpenSolaris: no aclocal
-  aclocal-1.11
-elif test -e /usr/bin/aclocal-1.10 ; then
-  aclocal-1.10
-else
-  aclocal
-fi
-
-#automake
-if test -e /usr/bin/automake-1.11 ; then
-  #OpenSolaris: no automake
-  automake-1.11 -a
-elif test -e /usr/bin/automake-1.10 ; then
-  automake-1.10 -a
-else
-  automake -a
+chmod a+x configure
+if [ -z "$NOCONFIGURE" ]; then
```

Comment:
I am reluctant to have default behavior to launch `./configure`, it is not the previous behavior and `autogen.sh` is planned for generating files for the final user, here we would have to change our build scripts in other places if we don't want to launch unnecessarily `./configure`.